clusterdock/topologies/cdh/actions.py

```diff
@@ -215,3 +281,110 @@ def remove_files(cluster, files, nodes):
     logger.info("Removing files (%s) from hosts (%s)...",
                 ', '.join(files), ', '.join([node.fqdn for node in nodes]))
     cluster.ssh('rm -rf {0}'.format(' '.join(files)), nodes=nodes)
+
+def _install_kdc_server(cluster, kdc_node, network, kerberos_principals):
```

Comment:
Just for readability, we can probably avoid using the underscore-leading function names throughout since `actions.py` is pretty much internal use only anyway.